docs/user-guide.md

```diff
@@ -61,6 +61,7 @@ Usage of kube-router:
       --ipvs-graceful-period duration                 The graceful period before removing destinations from IPVS services (e.g. '5s', '1m', '2h22m'). Must be greater than 0. (default 30s)
       --ipvs-graceful-termination                     Enables the experimental IPVS graceful terminaton capability
       --ipvs-sync-period duration                     The delay between ipvs config synchronizations (e.g. '5s', '1m', '2h22m'). Must be greater than 0. (default 5m0s)
+      --ipvs-permit-all                               Enables ipvs INPUT/CHAIN for traffic coming into nodes (default true)
```

Comment:
Can we keep it more user-friendly description without getting into implementation details like `INPUT/CHAIN` etc.

Something like `permit all inbound traffic to service VIP's`

Reply:
Yeah I'm not the best at describing these scenarios as I'm not super privvy to CNI terminology. 

Done! Thank you.